pkg/devspace/build/registry/util.go

```diff
@@ -67,13 +69,13 @@ func IsImageAvailableRemotely(ctx context.Context, imageName string) (bool, erro
 	return image != nil, nil
 }
 
-func CopyImageToRemote(ctx context.Context, imageName string, writer io.Writer) error {
+func CopyImageToRemote(ctx context.Context, client dockerclient.Client, imageName string, writer io.Writer) error {
 	ref, err := name.ParseReference(imageName)
 	if err != nil {
 		return err
 	}
 
-	image, err := daemon.Image(ref, daemon.WithContext(ctx))
+	image, err := daemon.Image(ref, daemon.WithContext(ctx), daemon.WithClient(client.DockerAPIClient()))
```

Comment:
This update allows the local registry push to work with either the local docker daemon, or the Minikube docker daemon.